src/main/cms/cms_menu_misc.c

```diff
@@ -149,8 +147,7 @@ static const void *cmsx_menuMiscOnExit(displayPort_t *pDisp, const OSD_Entry *se
     UNUSED(pDisp);
     UNUSED(self);
 
-    motorConfigMutable()->minthrottle = motorConfig_minthrottle;
-    motorConfigMutable()->digitalIdleOffsetValue = 10 * motorConfig_digitalIdleOffsetValue;
+    motorConfigMutable()->motorIdle = 10 * motorConfig_motorIdle;
```

Comment:
```suggestion
    motorConfigMutable()->motorIdle = motorConfig_motorIdle;
```

Reply:
As explained in the comment on like 138, this is to retain the existing OSD experience, where the user sets 55 if they want 5.5%.  It would be really annoying if the user had to go 551, 552, 553 etc ten times just to go from 5.50% to 5.60%, especially in an OSD.